run_all_manuscript_workflows.py

```diff
@@ -3,6 +3,15 @@
 from nuc_morph_analysis.analyses.colony_area import colony_area_workflow
 from nuc_morph_analysis.analyses.segmentation_model_validation import seg_model_validation_figure_workflow
 
+from nuc_morph_analysis.lib.preprocessing.global_dataset_filtering import load_dataset_with_features
```

Comment:
Do we want these changes to be the general behavior? I feel like they were just for a fast test...